grpc-proto/proto/schema.proto

```diff
@@ -85,6 +85,11 @@ message CqlKeyspace {
 
 message DescribeKeyspaceQuery {
   string keyspace_name = 1;
+  // The value of `CqlKeyspaceDescribe.hash` the last time the client described that keyspace.
+  // - if it matches the current hash on the server, the server will return an empty response, to
+  //   indicate that the client can continue using the last known value.
+  // - if it doesn't match, or is not provided, the server will return a regular response.
+  google.protobuf.Int32Value hash = 2;
```

Comment:
this is optional right?

Reply:
As per my earlier comment, I'd prefer 64-bit hash.

Reply:
Is `Int32Value` used to allow explicit "missing" vs defaulting to 0? That is, this is similar to `Integer` vs `int`.
Makes sense I guess and avoids having to have special handling for value `0`.

Reply:
Yes. It generate a `hasHash()` method that allows you to check if the value was explicitly set (vs a simple `sint32` that would default to 0 when not set).

Reply:
One more thing: use of `String.hashCode()` (or any Java specified `hashCode()`) is probably not a good idea if we want to allow non-Java clients. `String.hashCode()` is in theory just implementation detail (in practice it will never be changed as that'd break too much code), i.e. not specified and (theoretically) not reliably implementable on other platforms.
Use of externally defined standard hash is preferable as implementations exist for many languages.